integration-tests/tests/src/utils/select.ts

```diff
@@ -24,14 +24,14 @@ type PlatformValues<T> = Partial<Record<keyof KnownEnvironment, T>> & {
  * @param platformValues containing value for each targeted platform.
  * @returns the value for the platform the test is running on.
  */
```

Comment:
this change may not be necessary, I realised when doing some testing that if `default == undefined` this helper function won't work. But then I'd like ism input on this part as having `any` as return type is not necessarily the best..

Reply:
What do you mean by `default == undefined`? I am unsure how this affects `select`'s type. If this is about the automatic derivation of the type from given values, you can manually specify i.e. `select<String | undefined>({default: "test"}})` so there is no need for `any` as a return type.